src/komet/komet.py

```diff
@@ -47,7 +47,9 @@ def main() -> None:
         _exec_kast(program=args.program, backend=args.backend, output=args.output)
     elif args.command == 'test':
         wasm = Path(args.wasm.name) if args.wasm is not None else None
-        _exec_test(wasm=wasm, id=args.id)
+        if args.max_examples < 1:
+            raise ValueError(f'--max-examples must be a positive integer (greater than 0), given {args.max_examples}')
+        _exec_test(wasm=wasm, max_examples=args.max_examples, id=args.id)
```

Comment:
Is there a way to run unlimited examples? I think this option ends up getting sent to `hypothesis`, it might have support for supplying `0` or `-1` for that, you'll have to check.

Reply:
Hypothesis complains when it is less than 1:
```
hypothesis.errors.InvalidArgument: max_examples=-1 should be at least one. You can disable example generation with the `phases` setting instead.
```

See: https://github.com/HypothesisWorks/hypothesis/blob/d377752a2e98b86e65f49af86895c767a662f994/hypothesis-python/src/hypothesis/_settings.py#L366